src/test/scala/chiselTests/LayerSpec.scala

```diff
@@ -83,43 +89,48 @@ class LayerSpec extends ChiselFlatSpec with Utils with MatchesAndOmits {
 
   }
 
-  "Layers error checking" should "require that a nested layer definition matches its declaration nesting" in {
+  they should "allow to define additional layer-colored probes using enables" in {
 
     class Foo extends RawModule {
-      layer.block(A.B) {
-        val a = Wire(Bool())
+      // Without this enable, this circuit is illegal because `C` is NOT enabled
+      // when `A` is enabled.  Cf. tests checking errors of this later.
+      layer.enable(A)
```

Comment:
Is enable the global enable for this Module?

Reply:
Yes, it's indicating that this module has layer `A` enabled. This will emit:

```
public module Foo enablelayer A :
  ; ...
```

If there is more than one enable, then those will get appended, e.g., `layer.enable(A); layer.enable(B)` will be:

```
public module Foo enablelayer A enablelayer B :
  ; ...
```